### PR TITLE
Bloxawards.com

### PR DESCRIPTION
**Pull request by aviramdayan**
new script

### CODE REVIEW

File: Bloxawards.com.fc [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+        //utility.DoSolve();
+        if (result > 0)
+        {
+          return Fail("fail not solve"); //return to solve again - if something wrong
+        }
+
+        Wait();	
+	    Wait(2);
+		
+        var ButtonLog = ElementByXPath("/html/body/div/div[4]/div/div/form/button");
```
Comment (faucetcollector) [inline Bloxawards.com.fc]: Best would be to find a proper small xpath for this element

File: Bloxawards.com.fc [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+
+    string DomainName_String = "https://bloxawards.com/";
+
+    public override void Start()
+    {
+        //Title that shows in the browser. Is used to identify and close popup windows
+        Title = "Bloxawards.com";
+
+        //After we did try to claim on the faucet we search for these elements to determine if it was a success or a fail
+        SuccessXPath = "//*";
```
Comment (faucetcollector) [inline Bloxawards.com.fc]: No success/fail xpath?

File: Bloxawards.com.fc [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+
+
+
+
+    public override int DoSolveFaucet()
+    {
+    	
+        GoToUrl(DomainName_String+"/giveaway");
+        Wait();		
+        Wait(2);
```
Comment (faucetcollector) [inline Bloxawards.com.fc]: These two lines are the same as Wait(3);
or Wait(2, 3);

File: Bloxawards.com.fc (hunk 1 above)
Comment (aviramdayan) [reply Bloxawards.com.fc]: yeah for sure not i write this section, i will fix it

File: Bloxawards.com.fc (hunk 2 above)
Comment (aviramdayan) [reply Bloxawards.com.fc]: tahre is no success/fail xpath
so i did in the DoSolveFaucet protection

`
    public override int DoSolveFaucet()
    {
    	
        GoToUrl(DomainName_String+"/giveaway");
        Wait();		
        Wait(2);		
	
        var claimButton = ElementByClass("btn-main2");
		
        if (!IsVisible(claimButton))
        {
            //We couldn't find the button, return we have a failure.
            Log("Claim button not found or not visible.");
	    return GetWaitSetting(); //fail
        }
        
        var resultClaim = SolveCaptcha();
        if (resultClaim > 0)
        {
            //Unable to solve the captcha - return the failure
            return resultClaim;
        }
		
        if (IsVisible(claimButton))
        {
	    Log("claimButton - now");
            Click(claimButton);
        }	
		
		Wait(3);
		
	ad = true;    // success
        //return GetWaitSetting();	
		
        return base.DoSolveFaucet();
    }
`

File: Bloxawards.com.fc (hunk 3 above)
Comment (aviramdayan) [reply Bloxawards.com.fc]: Before he goes to the site, I always put in first Wait (); (Clean)
After it loads i give another Wait (2); for protection I found this very useful

**Pull request merged**